lightning/src/blinded_path/mod.rs

```diff
@@ -210,6 +210,30 @@ impl BlindedPath {
 			},
 		}
 	}
+
+	/// Attempts to a use a compact representation for the [`IntroductionNode`] by using a directed
+	/// short channel id from a channel in `network_graph` leading to the introduction node.
+	pub fn compact_introduction_node(&mut self, network_graph: &ReadOnlyNetworkGraph) {
```

Comment:
Pedantic but the convention is for setters to use the `set_` prefix: https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#gettersetter-apis and I think it would read a bit cleaner in this case.

Reply:
Hmmm... this is not a setter in the normal sense, though, as we don't pass a value to set a field with. Nor is the field private, which would necessitate a setter. It also may *not* update the field if a value cannot be found in the network graph. Happy to go with a better name. Just not sure if a `set_` prefix is appropriate here.